lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -586,9 +586,10 @@ module.exports = class MethodMappings {
 
         async sendKeysToElement(webElementOrId, value) {
           if (Array.isArray(value)) {
+            if (value.includes(undefined)){
+              throw new Error('sendKeys - each key must be a number or string');
+            }
```

Comment:
is this change necessary if selenium already handles this? why are we checking only when the value is of an array type

Reply:
selenium does not support the array type, that is provided by night watch. Hence we need to handle errors for the array type. Rest are handled by selenium:

https://github.com/SeleniumHQ/selenium/blob/468b5e345d430688776aa64f65fc2a1395e7aee5/javascript/node/selenium-webdriver/lib/webdriver.js#L2521

Reply:
I think this Line: https://github.com/SeleniumHQ/selenium/blob/468b5e345d430688776aa64f65fc2a1395e7aee5/javascript/node/selenium-webdriver/lib/webdriver.js#L2528 rasing the same error, hence I don't think that we need to throw an error from Nightwatch side?

Reply:
In case of an array, we join the elements to form a string. If one of those elements contains a the value `undefined` or `null` then the resulting string becomes "undefined". In that case selenium thinks that you are trying to type  `u n d e f i n e d` and does not throw any error. Which is wrong.

Reply:
I think it is missing a check for elements being `null`.

Reply:
Handled `null` case also [c5901a1](https://github.com/nightwatchjs/nightwatch/pull/3115/commits/c5901a1c34231ba872788b08ebc2a40e31d57e93)